common.sh

```diff
@@ -58,7 +58,7 @@ export SSH_PUB_KEY="${SSH_PUB_KEY:-$(cat $HOME/.ssh/id_rsa.pub)}"
 # mirror images for installation in restricted network
 export MIRROR_IMAGES=${MIRROR_IMAGES:-}
 
-export USE_IPV4=${USE_IPV4:-true}
+export USE_IPV4=${USE_IPV4:-false}
```

Comment:
Do we want to take this opportunity now to make this more versatile? e.g. IP_STACK=ipv6/ipv4/dualstack?

It'd avoid a change in the interface later when we have to do dual stack

Reply:
OK, sure, not a big deal